### PR TITLE
Refactor of Subscriptions

### LINKED ISSUE #767: Event Subscriptions input improvment
**Description**
As a user, I want to know what I write wrong and UI has to inform me what it's wrong and I want to have better informations from UI what I have to do.

**Expected result**
UI has to mark wrong fields in the creation modal.

**Actual result**
Validation is working but I don't know what is wrong, because the wrong fields are not merked.

**Acceptance criteria**
- [x] Rename Service to Sink
- [x] Add Text field with generated sink after Service/Sink dropdown
- [x] Rename placeholder `Choose a service for the sink`
- [x] Event Type can't be readOnly field -> change it on a text field or message strip information
- [x] Improve better informations for users in tooltips and placeholders
- [x] Remove the link from sink
- [x] Add Column `Controled by` with service and link to service

### PR DESCRIPTION
**Pull request by mrCherry97**
**Description**
As in the issue, a user has to be informed about bad inputs.
Changes proposed in this pull request:

- Renamed Event Subscription to Subscription
- Rename directories and paths from `Event Subscription` to `Subscription`
- Improve tooltips and placeholders
- Change input Event type in the Simple view to the MessageStrip
- Add Sink as a MesssageStrip

**Related issue(s)**
Closes #767

### CODE REVIEW

**Review by mmitoraj — DISMISSED** [review]

File: core-ui/src/components/Predefined/Create/Subscriptions/Subscriptions.create.js [hunk 1]
```diff
@@ -98,68 +97,83 @@ const SubscriptionsCreate = ({
...
       <ResourceForm.FormField
         required
-        label={t('services.name_singular')}
+        label={t('subscription.create.labels.service-name')}
         setValue={serviceName => {
           jp.value(
-            eventSubscription,
+            subscription,
             '$.spec.sink',
             `https://${serviceName}.${namespace}.svc.cluster.local`,
```
Comment (dariadomagala) [inline core-ui/src/components/Predefined/Create/Subscriptions/Subscriptions.create.js:158]:
```suggestion
            `http://${serviceName}.${namespace}.svc.cluster.local`,
```

**Review by dariadomagala — CHANGES_REQUESTED** [review]
* Why there is no validation error? 
* Event Type is a list so I believe it should be Event Type**s**
* I believe that on the Functions view, the dropdown with service chooser should be disabled. The subscription won't work for this function if we choose a different service

File: core-ui/src/components/Predefined/Create/Subscriptions/Subscriptions.create.js [hunk 2]
```diff
@@ -23,58 +23,57 @@ const SubscriptionsCreate = ({
...
-  const firstEventTypeValues = spreadEventType(firstEventType, eventTypePrefix);
+
+  const firstEventTypeValues = spreadEventType(
+    firstEventType,
+    DEFAULT_EVENT_TYPE_PREFIX,
+  );
 
   const notification = useNotification();
 
   useEffect(() => {
```
Comment (dariadomagala) [inline core-ui/src/components/Predefined/Create/Subscriptions/Subscriptions.create.js:54]: do we need this useEffect? wouldn't `setValue` that we have in line 154 be enough?

File: core-ui/src/components/Predefined/Create/Subscriptions/Subscriptions.create.js [hunk 3]
```diff
@@ -237,21 +242,17 @@ const SubscriptionsCreate = ({
...
-
       <TextArrayInput
         advanced
         required
         defaultOpen
-        tooltipContent={t('event-subscription.tooltips.event-type-advanced')}
+        tooltipContent={t('subscription.tooltips.event-type-advanced')}
         propertyPath="$.spec.filter.filters"
-        title={t('event-subscription.create.labels.event-type')}
+        title={t('subscription.create.labels.event-type')}
```
Comment (dariadomagala) [inline core-ui/src/components/Predefined/Create/Subscriptions/Subscriptions.create.js:255]: EventS typeS? In the advanced view, it's a list so it should be plural

**Review by dariadomagala — APPROVED** [review]

**Review by mmitoraj — APPROVED** [review]

**Pull request merged**